src/components/Table/filtering.ts

```diff
@@ -81,22 +102,23 @@ export const getSelectedFiltersList = <T extends TableRow>({
     const currentFieldFilters = selectedFilters[cur.accessor] || [];
     let orderedFilters: SelectedFiltersList = [];
 
-    if (currentFieldFilters.length) {
+    if (currentFieldFilters.selected?.length) {
```

Comment:
```suggestion
    if (currentFieldFilters.selected && currentFieldFilters.selected.length > 0) {
```

И тогда ниже в условии не надо будет перебивать TypeScript указанием, что selected уже существует.

Reply:
поправил